src/app/pages/StakingPage/Features/ValidatorList/index.tsx

```diff
@@ -51,7 +52,17 @@ export const ValidatorList = memo((props: Props) => {
   const columns: IDataTableColumn<Validator>[] = [
     {
       name: '',
-      cell: datum => <img src={process.env.PUBLIC_URL + '/logo192.png'} height="16px" width="16px" alt="" />,
+      cell: datum => (
+        <img
+          src={
+            datum.media?.logotype && isWebUri(datum.media.logotype)
+              ? datum.media.logotype
+              : process.env.PUBLIC_URL + '/logo192.png'
+          }
+          className={'logotype-small'}
```

Comment:
I'd add `loading="lazy"` (decent-enough browser support https://caniuse.com/loading-lazy-attr)

If all current validators had a logo:
- eager: 137 images get loaded immediately
- lazy: first 29 logos get loaded before scrolling (depends on cache, viewport height, logo sizes, connection speed?)

Reply:
Yeah, I tried that, but it doesn't work in react. It seems react fakes the window size and a complete `ValidatorList` table is loaded including all the images regardless of the `loading` img attribute.

Maybe we should do something like this in the future https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides/lazy-loading/ . So we load the original validator list in a lazy fashion :shrug:

Reply:
this is meant to be a browser native thing. why should 'react fakes the window size' affect it?

and since I'm commenting, that comparison of how many load is objectively true, but do we particularly want it? e.g. is there too much strain on the hosts of these logo images or something?

- eager: scrolling down shows logos right away
- lazy: blank cells when scrolling, logos show up later

Reply:
+1 it seemed to worked for me despite react :shrug: 

> but do we particularly want it?

No particularly good reason. It somewhat reduces bandwidth usage and strain on hosts (and maaaybe some images could timeout otherwise)